### ISSUE TITLE
bug: The output from the UpdateTable command does not include the SSEDescription

### ISSUE THREAD

**Issue by georgevanuta** [localstack/localstack#11781]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

The output from the UpdateTable command does not include the SSEDescription for a table which was created with it.

### Expected Behavior

The output from the UpdateTable command should include the SSEDescription when the table has it enabled.

### How are you starting LocalStack?

With a `docker run` command

#### How are you starting localstack (e.g., `bin/localstack` command, arguments, or `docker-compose.yml`)

    docker run localstack/localstack

#### Client commands (e.g., AWS SDK code snippet, or sequence of "awslocal" commands)

```bash
# launch localstack
# create a table with KMS
awslocal dynamodb create-table --table-name Example --attribute-definitions AttributeName=key,AttributeType=S  --key-schema AttributeName=key,KeyType=HASH  --provisioned-throughput ReadCapacityUnits=5,WriteCapacityUnits=5  --sse-specification Enabled=true,SSEType=KMS,KMSMasterKeyId=some-kms
# describe the table to see that it is KMS enabled
awslocal dynamodb describe-table --table-name Example
# update the table to PAY_PER_REQUEST
awslocal dynamodb update-table --table-name Example --billing-mode PAY_PER_REQUEST
# ^ the output from the update-table command does not include the SSEDescription
# sanity check to see the SSEDescription
awslocal dynamodb describe-table --table-name Example
```

### Environment

```markdown
LocalStack version: 3.8.2.dev70
LocalStack build date: 2024-10-31
LocalStack build git hash: 241bfbb8d
```

**Comment by IreneLime** [comment]
Hi @viren-nadkarni, may I work on this issue?

**Comment by viren-nadkarni** [comment]
Resolved with #11938
